src/std/fs.ab

```diff
@@ -101,7 +101,9 @@ pub fun file_glob_all(paths: [Text]): [Text]? {
         }
         combined = join(items, " ")
     }
-    let files = $ eval "for file in {combined}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done" $?
```

Comment:
I have no idea why `eval` was used before. It made me confusing, so I have stripped.